tests/production/DeleteUserSettingsFromCouchTests.js

```diff
@@ -64,7 +64,7 @@ gpii.tests.productionConfigTesting.getPrefsSafeDoc = function (prefsSafeRequest,
         prefsSafeRequest.options.path = "/gpii/prefsSafe-" + prefsSafeRequest.options.gpiiKey;
         fluid.log("No Prefs Safe to retrieve for GPII key " + prefsSafeRequest.options.gpiiKey);
     }
-    prefsSafeRequest.send(null, { port: 5984 });
+    prefsSafeRequest.send(null, { port: gpii.tests.productionConfigTesting.couchdbUrl.port });
```

Comment:
As `port: gpii.tests.productionConfigTesting.couchdbUrl.port` has been specified in the definition of requests, such as [here](https://github.com/klown/universal/blob/GPII-3832/tests/production/DeleteUserSettingsFromCouchTests.js#L251), is it still necessary to pass it at calling send()?

Reply:
Yes, it is necessary.  This issue came up previously for GPII-3333 and was discussed in that JIRA's related [pull request](https://github.com/GPII/universal/pull/718/files#r266101132).